src/components/Registration/HelpContactUs.vue

```diff
@@ -31,11 +31,19 @@
           </h2>
         </header>
 
-        <p class="my-4">
-          If you require further assistance adding a business or corporation, please contact us.
-        </p>
+        <div v-if="isTypeFirm">
+          <p class="my-4">
+            If you require further assistance adding a business, please contact us.
+          </p>
+          <FirmContactInfo />
+        </div>
 
-        <RegistriesContactInfo />
+        <div v-else>
```

Comment:
This "else" case will never be hit, because this component is used for registrations, which are only for firms.

Reply:
LGTM